### DISCUSSION TITLE
кто подскажет как обновить все майнеры?

### DISCUSSION THREAD

**Discussion by 9301039** [angelbbs/NiceHashMinerLegacy#197] [Show and tell]
и еще почему-то SRB только в CPU и его нет в списке GPU майнеров.. с чем может быть связано и как исправить?

**Comment by angelbbs** [comment]
1. Майнеры находятся в папке Miners программы. Можно обновлять вручную.
2. В программе SRB-Miner работает только с CPU и AMD GPU
